### PR TITLE
Add default attributes handling

### LINKED ISSUE #15: Support the service.name in the payload
At this moment, it seems the library does not have any method to set the `service.name` attribute. This is crucial since Honeycomb does several reports / query that involves the service name, and we need to know the lifespan of a request across multiple services, for example hitting API in the `api` application , then querying DB in the `academy` application etc.....

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
Fixes #15 (somewhat) - the service.name attribute is reserved for all horizontal instances of the same binary (therefore it cannot be changed), but with this in place you can add `[{"component", "api"}]` to each specific trace provider.

FYI @onimsha

Signed-off-by: Maciej Szlosarczyk <maciej@mindvalley.com>

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **maciej-szlosarczyk** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by antonsatin — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 2bf8a00f399c7b21edb4e4ab20d10d167e962b93</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/20#pullrequestreview-854390096" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
